packages/web3-typescript-typings/CHANGELOG.md

```diff
@@ -3,3 +3,4 @@
 ## v0.9.3 - _January 11, 2018_
 
 * Add type for getData on a contract
+* Fixed the `defaultAccount` not allowing for `undefined` value
```

Comment:
Please add a PR number as we have here: https://github.com/0xProject/0x.js/blob/development/packages/0x.js/CHANGELOG.md